packages/clay-panel/src/index.tsx

```diff
@@ -135,6 +151,7 @@ function ClayPanel({
 			{collapsable && (
 				<>
 					<ClayButton
+						aria-controls={`panel-index-${indexOfElement}`}
```

Comment:
Hey @AkramYoussoufi can you look at other instances of how we generate an ID for aria-controls and copy that pattern?